### ISSUE TITLE
`external_location` ignored when `table_type='hive'`

### ISSUE THREAD

**Issue by TassOlympus** [dbt-athena/dbt-athena#539]
Using the latest version with a model that has a hive table type used to allow the specification and enforcement of the `external_location` configuration within the models, it appears it is now being ignored in the latest version when new hive tables are created

**Comment by nicor88** [comment]
@TassOlympus do you use a workgroup with enforced "configurations"? If that's the case external_location is fully ignored.
Also there was to change on how **external_location** is used in 1.7.0.

**Comment by TassOlympus** [comment]
@nicor88 there is a location specified in the work group being used but specifying "external_location" works for iceberg tables regardless of the workgroup location, it's just hive tables where it is ignored. I would imagine that should a location be specified at the model level that it would override the workgroup location? Is there a reason why it is different across table types?

**Comment by nicor88** [comment]
Iceberg table are managed tables, therefore the workgroup location is not used.
Hive tables are the contrary are not managed table, therefore what happen is that your "workgroup" enforced location is used by default.
You have 2 options:
- disable workgroup config enforcement in your workgroup setting
- if you cannot follow the first option, consider to create a new workgroup (whithout location set) only for dbt

**Comment by TassOlympus** [comment]
Thanks @nicor88, this is useful and helped resolve our issue (we went for the separate work group approach), could documentation perhaps be updated to articulate this information under https://github.com/dbt-athena/dbt-athena#table-location? The note was misleading since it did not apply to iceberg tables and seems to speak directly to "s3_data_naming" alone

**Comment by nicor88** [comment]
@TassOlympus good catch, docs must be updated indeed. Feel free to submit a PR to address this.
